CHANGELOG.md

```diff
@@ -17,6 +17,140 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Deprecated
 
```

Comment:
Something odd has happened with the changelog. I think it will be solved when we "update branch" and fix the conflicts. Let me know if you'd like me to help with this.